server/src/main/scala/com/cloudera/livy/server/interactive/CreateInteractiveRequest.scala

```diff
@@ -36,5 +36,6 @@ class CreateInteractiveRequest {
   var queue: Option[String] = None
   var name: Option[String] = None
   var conf: Map[String, String] = Map()
+  var scalaVersion: Option[String] = None
```

Comment:
> This field and per-session based Scala version check may not be useful currently, since SPARK_HOME is still a global setting. But as I mentioned this is one task for Livy to support different Sparks.

Since it's not useful currently, I would prefer to not expose this in CreateInteractiveSession and remain a global flag.